version.php

```diff
@@ -28,7 +28,7 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 452;
+$v_database = 453;
```

Comment:
this will cause conflict in my PR. Close and I'll bump to appropriate version. thanks